assets/article/plugins/style/style.js

```diff
@@ -50,6 +50,10 @@ ArticleEditor.add('plugin', 'style', {
                     title: 'Insert the HIGHLIGHTED PARAGRAPH (DIVIDER) class',
                     classname: 'highlighted-paragraph'
                 },
```

Comment:
You are using the same class for the gray column and the Highlighted paragraph: 

```
                'highlighted-paragraph': {
                    title: 'Insert the HIGHLIGHTED PARAGRAPH (DIVIDER) class',
                    classname: 'highlighted-paragraph'
                },
                'gray-column': {
                    title: 'Insert the GRAY COLUMN class',
                    classname: 'highlighted-paragraph'
                },
```

Also, please call it a grey card, at least for the users, if not in code